src/com/sap/piper/WhitesourceConfigurationHelper.groovy

```diff
@@ -79,6 +79,11 @@ class WhitesourceConfigurationHelper implements Serializable {
                     [name: 'excludes', value: '**/*sources.jar **/*javadoc.jar']
                 ]
                 break
+            case 'dub':
+                mapping += [
+                    [name: 'includes', value: '**/*.d **/*.di']
```

Comment:
Wouldn't we need a specific resolver to be enabled? Since to avoid any noise first all resolvers are disabled and therefore the specific one(s) for the scenario need to be enabled again. Sorry for not being more specific but I am lacking any experience on Dlang projects.

Reply:
@nevskrem at the moment there is no specific solver for DLang but D  is supported using the generic WhiteSource scan engine.